tests/rptest/tests/double_bytes_sent_test.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+import random
+from ducktape.mark.resource import cluster
+from ducktape.utils.util import wait_until
+
+from rptest.services.rpk_producer import RpkProducer
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.clients.types import TopicSpec
+
+
+class BytesSentTest(RedpandaTest):
+    """
+    Produce and consume some data then confirm the bytes sent metric.
+    """
+    topics = (TopicSpec(replication_factor=1), )
+
+    def __init__(self, test_context):
+        super(BytesSentTest, self).__init__(test_context=test_context,
+                                            num_brokers=1)
+
+        self._producer = RpkProducer(test_context,
+                                     self.redpanda,
+                                     self.topic,
+                                     msg_size=16384,
```

Comment:
I expected for the number of bytes sent to increase by 16384 after the producer finished but that is not the case. Instead the number of bytes sent increased by 535. Why?